learning-programming.md

```diff
@@ -78,7 +78,7 @@ The next thing to learn, is how to apply these general patterns to solve problem
 
 Let me illustrate with an example:
 
-Given the following problem _Write every number between 0 and 100 inclusive._ its solution in JavaScript would look as follows:
+Given the following problem _Write every number between 0 and 100 inclusive._ it's solution in JavaScript would look as follows:
```

Comment:
"its" is a possesive pronoun and `it's` is an abbrevation of `it is`, so `its` was correct here